src/manifest.ts

```diff
@@ -198,9 +198,13 @@ export interface SentenceCasePluginConfig extends ConfigurablePluginType {
 export interface WorkspacePluginConfig extends ConfigurablePluginType {
   merge?: boolean;
 }
+export interface GroupPriorityPluginConfig extends ConfigurablePluginType {
+  groups: string[];
+}
 export type PluginType =
   | DirectPluginType
   | ConfigurablePluginType
+  | GroupPriorityPluginConfig
```

Comment:
In general, I think it would be worth thinking about an eventual refactor that allows us to add additional plugins, options, etc., updating less lists of types, factories, etc.

On the positive, I like that the manifest + plugin based approach requires less updates to add new features, than adding new command line flags.